waku/message_test.go

```diff
@@ -495,3 +494,15 @@ func TestValidateAndParseSizeOfPayloadSize(t *testing.T) {
 		})
 	}
 }
+
+func TestEncodeDecodeVersionedResponse(t *testing.T) {
+	response := NewMessagesResponse(common.Hash{1}, []EnvelopeError{{Code: 1}})
+	b, err := rlp.EncodeToBytes(response)
+	require.NoError(t, err)
+
+	var mresponse MultiVersionResponse
+	require.NoError(t, rlp.DecodeBytes(b, &mresponse))
```

Comment:
I've also changed `bytes` to `b` to remove the type name / variable name collision.